ws-nextjs-app/pages/[service]/av-embeds/avembeds.stories.tsx

```diff
@@ -1,16 +1,143 @@
 import React from 'react';
+import { MediaBlock } from '#app/components/MediaLoader/types';
 import AvEmbedsPage from './AvEmbedsPageLayout';
-import parseAvRoute from './parseAvRoute';
 
-const Component = ({ requestUrl }: { requestUrl: string }) => {
-  const parsedAvRoute = parseAvRoute(requestUrl);
-
-  const fixtureData = {
-    input: requestUrl,
-    output: parsedAvRoute,
-  };
+const fixtureData = {
```

Comment:
Would it make sense to import one of the fixtures from above to use in storybook? Or maybe move this to a separate file & import it?